### ISSUE TITLE
rpmlint should warn about directories without X bit

### ISSUE THREAD

**Issue by darix** [rpm-software-management/rpmlint#1196]
probably hard fail would be even better.

**Comment by danigm** [comment]
Right now there's a error when the directory has no standard permissions, just here:

https://github.com/rpm-software-management/rpmlint/blob/main/rpmlint/checks/FilesCheck.py#L913-L914

```
        if perm != 0o755:
            self.output.add_info('E', pkg, 'non-standard-dir-perm', fname, '%o' % perm)
```

**Comment by danigm** [comment]
I've created a [new PR](https://github.com/rpm-software-management/rpmlint/pull/1199) with a couple of tests for this check. I've added the same binary package and in the testing environment it's showing the error, so the check is working correctly here.

In OBS it's not showing this error because it's automatically [filtered in opensuse](https://github.com/rpm-software-management/rpmlint/blob/main/configs/openSUSE/opensuse.toml#L171)
